content/api_chef_server.md

```diff
@@ -2604,6 +2606,184 @@ The response is similar to:
 </tbody>
 </table>
 
+### POST
+
+The `POST` method is used to create a container.
+
+This method has no parameters.
+
+**Request**
+
+``` none
+POST /organizations/NAME
+```
+
+This method has a request body similar to:
+
+``` javascript
+{
+  "containername": "mycontainer",
+  "containerpath": "mycontainer"
+}
+```
+
+**Response**
+
+The response is similar to:
+
+``` javascript
+{
+  "uri"": "https://chef.example/organizations/test/containers/mycontainer"
+}
+```
+
+**Response Codes**
+
+<table>
+<colgroup>
+<col style="width: 20%" />
+<col style="width: 80%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th>Response Code</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td><code>201</code></td>
+<td>OK. The request was successful.</td>
+</tr>
+<tr class="even">
+<td><code>401</code></td>
+<td>Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name, and that the correct key was used to sign the request.</td>
+</tr>
+<tr class="odd">
+<td><code>403</code></td>
+<td>Forbidden. The user who made the request is not authorized to perform the action.</td>
+</tr>
+<tr class="even">
+<td><code>404</code></td>
+<td>Not found. The requested object does not exist.</td>
+</tr>
+</tbody>
+</table>
+
+/containers/NAME
+----------------
+
+The `/containers/Name` endpoint has the following methods: `DELETE`, `GET`.
+
+### DELETE
+
+The `DELETE` method is used to remove a container.
+
```

Comment:
Add here something like `NOTE: while this method is implemented, it is not supported, and use will likely break your system, or not have the effect you expect.`

Reply:
In the queue. Thanks